apps/services/handlers.py

```diff
@@ -102,10 +146,26 @@ async def read_user_request_booking_hostel(university_id: int, user=Depends(get_
     }
 
 
-@services_router.put("/{university_id}/user-request/{user_request_id}", response_model=JSENDOutSchema[CancelRequestOut],
+@services_router.put("/{university_id}/user-request/{user_request_id}",
+                     name="update_cancel_user_request",
```

Comment:
why update?

Reply:
CRUD, that's why. Of course, I can change, but only one (or two) will differ from others. What to do?

Reply:
ok, for now it's ok